fix: wizard minimum height to avoid hidden elements

re2005 opened the pull request:
Related to issue:
https://github.com/legalthings/legaldocx/issues/1137

re2005 committed:
fix: wizard minimum height to avoid hidden elements

jasny reviewed (changes requested):
This has `min-width` instead of `min-height`.
Can you use `%` or `vh`?

re2005 committed:
fix: should be min-height

jasny commented on css/general.css:
```diff
@@ -4,6 +4,7 @@
 
 .wizard {
     padding-right: 25px;
+    min-height: 600px;
```
`min-height` of 600px seems very arbitrary. For bigger screens, the date picker will still be partially hidden.

re2005 committed:
refactor: remove overflow hidden

Since all children wrapping elements have already overflow hidden it may be not necessary.
